README.md

```diff
@@ -172,6 +174,41 @@ data:
     OVN_LOG_LEVEL=dbg
 ```
 
+### Configuring OVNKubernetes On a Hybrid Cluster
+OVNKubernetes supports a hybrid cluster of both Linux and Windows nodes on x86_64 hosts. The ovn configuration is done as described above. In addition the `hybridOverlayConfig` can be included as follows:
+
+Create the cluster using openshift-install. First get the install-config. Use a convenient directory for the cluster.
+```
+$ openshift-install --dir=MY_CLUSTER create install-config
+```
+Edit the MY_CLUSTER/install-config and change the `networkType:` to OVNKubernetes
+
+Next create the manifests:
+```
+$ openshift-install --dir=MY_CLUSTER create manifests
+$ cp MY_CLUSTER/manifests/cluster-network-02-config.yml MY_CLUSTER/manifests/cluster-network-03-config.yml
+```
+Edit the new file as follows:
+- change first line `apiVersion: config.openshift.io/v1` to `apiVersion: operator.openshift.io/v1`
+- Add the following to the `spec:` section
+
+```yaml
+spec:
+  defaultNetwork:
+    type: OVNKubernetes
+    ovnKubernetesConfig:
+      hybridOverlayConfig:
+        hybridClusterNetwork:
+        - cidr: 10.132.0.0/14
+          hostPrefix: 23
+```
+The hybridClusterNetwork cidr and hostPrefix are used when adding windows nodes. This cidr must not overlap the ClusterNetwork cidr.
```

Comment:
@pecameron windows -> Windows?

Reply:
Put the first cidr here in backticks to show it's a literal. Capitalize the second I think because it's a generic reference.

Reply:
@dcbw Did I get the changes right? 
Also, found that we are limited to a single hybridClusterNetwork: cidr.
Question: did we change from config.openshift.io/v1 to operator.openshift.io/v1? If so installer is wrong.

Reply:
@pecameron we shouldn't be limited to a single hybrid overlay CIDR. For the YAML I think it would be:

```
    ovnKubernetesConfig:
      hybridOverlayConfig:
        hybridClusterNetwork:
        - cidr: 10.132.0.0/14
          hostPrefix: 23
        - cidr: 11.150.0.0/14
          hostPrefix: 23
```

Reply:
@dcbw The code does 1 CIDR.
pkg/network/ovn_kubernetes.go:82:		data.Data["OVNHybridOverlayNetCIDR"] = c.HybridOverlayConfig.HybridClusterNetwork[0].CIDR

Reply:
@pecameron I guess we should fix that in CNO then... @JacobTanenbaum can you take that task?